src/main/java/frc/robot/subsystems/CommandFactory.java

```diff
@@ -214,31 +214,51 @@ public Command algaeGoToL3() {
         .until(m_algaeRollers.hasAlgae());
   }
 
-  public Command algaeGoToL4() {
-    Command elevator = m_elevator.setCoralPosition(CoralLevels.L4);
+  public Command dealegfyL3() {
+    Command elevator = m_elevator.setCoralPosition(CoralLevels.DEALGEFY_L3);
     Command pivotAlgaeRollers = m_algaePivot.goToLowerSetpoint();
     Command IntakeAlgae = m_algaeRollers.feedIn();
 
     return elevator.andThen(pivotAlgaeRollers).alongWith(IntakeAlgae);
   }
 
+  public Command placeCoralL4() {
```

Comment:
Remove these placeCoralL4, placeCoralL3 and placeCoralL2 and replace with one reusable command for placing on a level:
```
  public Command placeCoral(CoralLevels level) {
    Command pivotCoralRollersCommand = m_coralPivot.goToLowerSetpoint();
    Command ReleaseCoral = m_coralRollers.rollOutCommand(level);

    return pivotCoralRollersCommand.andThen(ReleaseCoral);
  }
```